### PR TITLE
Release 5.6.0

### PR DESCRIPTION
**Pull request by tadgh**
Merge back of 5.6.0 post-release.

### CODE REVIEW

File: hapi-fhir-validation/src/main/java/org/hl7/fhir/common/hapi/validation/support/RemoteTerminologyServiceValidationSupport.java [hunk 1]
```diff
@@ -143,6 +142,167 @@ public IBaseResource fetchCodeSystem(String theSystem) {
...
+					break;
+				case "abstract":
+					result.setCodeIsAbstract(((parameterComponent.getValue() == null)?false:Boolean.parseBoolean(parameterComponent.getValue().toString())));
+					break;
+			}
+		}
+		return result;
+	}
+
+	private LookupCodeResult generateLookupCodeResultR4(String theCode, String theSystem, org.hl7.fhir.r4.model.Parameters outcomeR4) {
```
Comment (jdar8) [inline hapi-fhir-validation/src/main/java/org/hl7/fhir/common/hapi/validation/support/RemoteTerminologyServiceValidationSupport.java:247]: Does outcomeR4 need to be named theOutcomeR4

File: hapi-fhir-validation/src/main/java/org/hl7/fhir/common/hapi/validation/support/RemoteTerminologyServiceValidationSupport.java [hunk 2]
```diff
@@ -143,6 +142,167 @@ public IBaseResource fetchCodeSystem(String theSystem) {
...
+				}
+				break;
+			default:
+				throw new UnsupportedOperationException("Unsupported FHIR version '" + fhirVersion.getFhirVersionString() +
+					"'. Only DSTU3 and R4 are supported.");
+		}
+		return null;
+	}
+
+	private LookupCodeResult generateLookupCodeResultDSTU3(String theCode, String theSystem, org.hl7.fhir.dstu3.model.Parameters outcomeDSTU3) {
```
Comment (jdar8) [inline hapi-fhir-validation/src/main/java/org/hl7/fhir/common/hapi/validation/support/RemoteTerminologyServiceValidationSupport.java:188]: Does outcomeDSTU3 need to be named theOutcomeDSTU3

**Review by jdar8 — COMMENTED** [review]
I love the level of docs and comments!

I don't know if some of the parameters need to be renamed or not?

File: hapi-fhir-validation/src/main/java/org/hl7/fhir/common/hapi/validation/support/RemoteTerminologyServiceValidationSupport.java (hunk 1 above)
Comment (tadgh) [reply hapi-fhir-validation/src/main/java/org/hl7/fhir/common/hapi/validation/support/RemoteTerminologyServiceValidationSupport.java:247]: it ought to be yup

**Review by jdar8 — APPROVED** [review]

**Pull request merged**